### PR TITLE
update endpointing API

### PR DESCRIPTION
**Pull request by shirgoldbird**

### CODE REVIEW

File: developers/src/pages/documentation/guides/understand-endpointing-interim-results/index.mdx [hunk 1]
```diff
@@ -30,6 +31,16 @@ When Deepgram identifies that an endpoint has been reached, the response is mark
 
 As an example, consider a speaker responding "yes" on a live call. Although Deepgram's default algorithm automatically sends final results every few seconds, you might want your system to react to the "yes" immediately. In this case, Endpointing would be useful because it would prompt Deepgram to detect the pause after "yes" and quickly send back a finalized result (marked with `speech_final: true`).
 
+### Controlling Endpointing
+
+By default, Deepgram identifies an endpoint after 10 milliseconds (ms) of silence. You can adjust this length of time. To do so, send the `endpointing` parameter in the request to the Deepgram API to the value you would like to use.
+
+Some examples of use cases for which you might want to adjust `endpointing` include: 
+
+- A virtual agent or chatbot that expects users to speak in short utterances and needs to engage with them as soon as they stop speaking. In this case, you would likely want to choose a low value for `vad_turnoff` or leave it as the default of 10 ms.
```
Comment (jviso) [inline developers/src/pages/documentation/guides/understand-endpointing-interim-results/index.mdx:40]: I think you want to update `vad_turnoff` to be `endpointing` here and on line 42.

**Review by jviso — COMMENTED** [review]
We can deploy the API change before EOW. I'll aim for Thursday. We should wait to deploy the docs change until the API change is live.

File: developers/src/pages/documentation/guides/understand-endpointing-interim-results/index.mdx (hunk 1 above)
Comment (shirgoldbird) [reply developers/src/pages/documentation/guides/understand-endpointing-interim-results/index.mdx:40]: Thanks! Updated.

**Comment by shirgoldbird** [comment]
@jviso can you approve so we can release now that the changes are out?

**Review by jpvajda — APPROVED** [review]

**Pull request merged**